### ISSUE TITLE
Create a calculator sample

### ISSUE THREAD

**Issue by sebastianbenz** [ampproject/amp-by-example#1575]
Similar to https://www.creditkarma.com/calculators/debtrepayment. Features we should demonstrate:

* client-side calculator with amp-bind
* focus on results
* charts (nice to have)
* server-side implementation using amp-form (needed ?)

//cc @ericlindley-g

**Comment by kul3r4** [comment]
Nice, so server-side implementation using amp-form would be an alternative to calculating the the numbers on client side?

**Comment by sebastianbenz** [comment]
Yes, server-side would do the same thing. I'm not sure though whether we really need to include it into the sample.
